config.yml.sample

```diff
@@ -32,7 +32,11 @@ hlp:
     login_url: http://www.hard-light.net/forums/index.php?action=login2
 
 nightly:
-    hlp_board: 92
+    hlp_board: 173
+
+release:
+    hlp_board: 50
 
 templates:
-    nightly: templates/nightly.mako
\ No newline at end of file
+    nightly: /media/diverses/Code/Sources/linux/asarium/nightlybuild/templates/nightly.mako
```

Comment:
Will these lines have to be changed to full paths going forward or will leaving them as relative paths in the used config be ok?